src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -1,146 +1,214 @@
 <template>
   <section id="people-and-roles">
-    <v-card flat>
-      <!-- Header -->
-      <div class="section-container header-container">
-        <v-icon color="appDkBlue">mdi-account-multiple-plus</v-icon>
-        <label class="font-weight-bold pl-2">{{ orgPersonLabel }}</label>
-      </div>
-
-      <!-- Instructional people and roles text (base corrections only)-->
-      <article v-if="isBenBcCccUlcCorrectionFiling" class="section-container">
-        This application must include the following:
+    <!-- Restoration conversion and extension add buttons -->
+    <div v-if="(isLimitedConversionRestorationFiling || isLimitedExtendRestorationFiling)">
+      <article>
+        <header>
```

Comment:
@jonathan-longe Ketaki and I discussed this earlier today and I approve this "duplication".

Part of this is due to the different display style needed, and part of it is because of the different applicant rules, between filings.

The alternative would be a wrapper to handle the style differences, and still some conditionals (possibly resource driven) to handle the rest. It still wouldn't be super-clean, and we don't know what the future looks like because you are still designing it :)

What do you think -- is this OK for now?

Reply:
@severinbeauvais and @ketaki-deodhar I hear you.  Let's leave this as is for now.  When we have some time for refactoring, I would suggest we create components for each filing type rather than adding `v-if="filingTypeXXX`.  Extending `PeopleAndRoles` common component to handle every filing type makes the common component more complicated and more difficult to read.  Worse, when we modify a common component to handle a new filing we risk breaking existing functionality that may not get tested.

Reply:
Refactoring this will be an interesting exercise.

In Create UI, there are separate PeopleAndRoles components (for different filing types) but a common mixin as they share a lot of code. That was pretty easy to work on, so maybe that's a pattern we could follow here (future).